source/types/uuid.go

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"fmt"
+
+	"github.com/jackc/pgx/v5/pgtype"
+)
+
+type UUIDFormatter struct{}
+
+// Format returns:
+// * string format of Time when connectorn is not builtin
+// * time type in UTC when connector is builtin
+func (UUIDFormatter) Format(v any) (string, error) {
+	b, ok := v.([16]byte)
```

Comment:
So a UUID can come in either as a byte slice (with type set to pgtype.UUIDOID) or as a string (what's the type in that case?)?